doc/assets.md

```diff
@@ -10,41 +10,37 @@ For assets, a special **`ezdesign` asset package** is available.
 <img src="{{ asset("images/foo.png", "ezdesign") }}" alt="foo"/>
 ```
 
-Using `ezdesign` package will resolve current design with theme fallback.
+Using the `ezdesign` package will resolve current design with theme fallback.
 
 By convention, an asset theme directory can be located in:
-* `<bundle_directory>/Resources/public/themes/`
-* `web/assets/themes/`
+- `<bundle_directory>/Resources/public/themes/`
+- `web/assets/themes/`
 
 Typical paths can be for example:
-* `<bundle_directory>/Resources/public/themes/foo/` => Assets will be part of `foo` theme.
-* `<bundle_directory>/Resources/public/themes/bar/` => Assets will be part of `bar` theme.
-* `web/assets/themes/biz/` => Assets will be part of `biz` theme.
+- `<bundle_directory>/Resources/public/themes/foo/` => Assets will be part of the `foo` theme.
+- `<bundle_directory>/Resources/public/themes/bar/` => Assets will be part of the `bar` theme.
+- `web/assets/themes/biz/` => Assets will be part of the `biz` theme.
 
-It is also possible to use `web/assets` as a global override directory. 
-If called asset is present **directly under this directory**, it will always be considered first.
+It is also possible to use `web/assets` as a global override directory. If called asset is present **directly under this directory**, it will always be considered first.
 
-> **Important**: You must have *installed* your assets with `assets:install` command, so that your public resources are
-> *installed* into the `web/` directory.
+> **Important**: You must have *installed* your assets with `assets:install` command, so that your public resources are *installed* into the `web/` directory.
 
 ## Fallback order
-Default fallback order is the following:
-* Application assets directory: `web/assets/`
-* Application theme directory: `web/assets/themes/<theme_name>/`
-* Bundle theme directory: `web/bundles/<bundle_directory>/themes/<theme_name>/`
+
+The default fallback order is:
+- Application assets directory: `web/assets/`
+- Application theme directory: `web/assets/themes/<theme_name>/`
+- Bundle theme directory: `web/bundles/<bundle_directory>/themes/<theme_name>/`
 
 Calling `asset("js/foo.js", "ezdesign")` can for example be resolved to `web/bundles/app/themes/my_theme/js/foo.js`.
 
-## Performance & assets resolution
-When using themes, paths for assets are resolved at runtime. This is due to how Symfony Asset component is integrated
```

Comment:
> Breaking the lines at 100-120 may seem nice and tidy from developer point of view, but this is documentation, and we prefer to concern about our reader to get proper content fast - we can scroll a few inches to the right if that's what it takes.

No. The fact that many break lines inside bullets breaks something doesn't mean that we won't use breaklines.

The point of markdown is to be humanly readable. Auto-wrapping of lines is something developers never do.

The point of having this_developer documentation_ as markdown is that _developers_ can contribute to it.

> Making changes in such "chopped" content is also a pain to do review later. Instead of 3-4 lines to check, reviewer would have to check dozens of lines and make sure that everything is clean and tidy

Wrong. It makes it harder. The human brain is built to process short chunks of text, not long lines. Each change in the same line will move the comparison left or right, making it harder to review the longer the line is. Most review tools do support side-by-side reviews, in which case long lines require permanent left-right scrolling.

A line break does not create a line break in the rendered result. A new line does. It's a _feature_ of markdown.